test/common/testlib.py

```diff
@@ -204,7 +204,10 @@ def go(self, hash, host="localhost"):
         self.call_js_func('ph_go', hash)
 
     def mouse(self, selector, type, x=0, y=0, btn=0, force=False):
-        self.wait_visible(selector)
+        if force:
+            self.wait_present(selector)
```

Comment:
That commit should be squashed into yours as before your commit it needed to introduce visibility change and after your commit it would be fixing issues in your commit. I squashed it and explained it in your commit.